src/azure-cli/azure/cli/command_modules/vm/tests/latest/test_vm_commands.py

```diff
@@ -6491,7 +6491,7 @@ def test_vmss_simple_orchestration_mode(self, resource_group):
 
         self.cmd('ppg create -g {rg} -n {ppg}')
         self.cmd('vmss create -g {rg} -n {vmss} --orchestration-mode Flexible --single-placement-group false '
-                 '--ppg {ppg} --platform-fault-domain-count 3 --admin-username vmtest',
+                 '--ppg {ppg} --platform-fault-domain-count 3',
```

Comment:
Fix the following errors:
> {"status":"Failed","error":{"code":"DeploymentFailed","message":"At least one resource deployment operation failed. Please list deployment operations for details. Please see https://aka.ms/DeployOperations for usage details.","details":[{"code":"BadRequest","message":"{\r\n  \"error\": {\r\n    \"code\": \"BadRequest\",\r\n    \"message\": \"Virtual Machine Scale Set with orchestration mode 'Flexible' and a Virtual Machine profile must use managed disks.\"\r\n  }\r\n}"}]}}

`--admin-username vmtest` was added from PR #21761 by mistake. It will result in adding redundant VM profile when creating an empty VMSS